lib/bundler/cli.rb

```diff
@@ -164,28 +168,28 @@ def check
       "Specify the number of jobs to run in parallel"
     method_option "local", :type => :boolean, :banner =>
       "Do not attempt to fetch gems remotely and use the gem cache instead"
-    method_option "no-cache", :type => :boolean, :banner =>
+    deprecated_option "no-cache", :type => :boolean, :banner =>
       "Don't update the existing gem cache."
     method_option "force", :type => :boolean, :banner =>
       "Force downloading every gem."
     method_option "no-prune", :type => :boolean, :banner =>
```

Comment:
the 2.0 cache command is the 1.0 package command... it basically runs `install && Bundler.load.cache` and it already has a `--no-prune` flag. unrelated, but this reminds me we need to default `cache_all_platforms` to true in 2.0